src/ui/map.js

```diff
@@ -2761,6 +2773,55 @@ class Map extends Camera {
         return this;
     }
 
+    /**
+     * Update the average visible elevation by sampling terrain
+     *
+     * @returns {boolean} true if elevation has changed from the last sampling
+     */
+    _updateAverageElevation(timeStamp: number): boolean {
+        if (timeStamp === undefined) throw new Error('undefined time!');
```

Comment:
maybe replace with an assert?

Reply:
Oh, oops! That was a debug statement left over from trying to catch where bad data was sneaking (it was in the cameraWorldSize getter).